src/main/java/org/codehaus/mojo/versions/api/DefaultVersionsHelper.java

```diff
@@ -895,6 +895,35 @@ else if ( !excludePropertiesList.isEmpty() && excludePropertiesList.contains( pr
         return propertyVersions;
     }
 
+    @Override
+    public Dependency interpolateVersion( final Dependency dependency, final MavenProject project )
+    {
+
+        // resolve version from model properties if necessary (e.g. "${mycomponent.myversion}"
+        if ( dependency.getVersion().startsWith( "${" ) )
+        {
+            final String resolvedVersion = project.getOriginalModel()
+                .getProperties().getProperty(
+                    dependency.getVersion().substring( 2, dependency.getVersion().length() - 1 ) );
+            if ( resolvedVersion != null && !resolvedVersion.isEmpty() )
+            {
+                dependency.setVersion( resolvedVersion );
+            }
+        }
+        return dependency;
+    }
+
+    private List<String> getSplitProperties( String commaSeparatedProperties )
```

Comment:
Where this new private method is used?

Reply:
Not from me. Perhaps came from the rebase? Should I remove it?

Reply:
Please don't remove it. It's being used in AbstractVersionsHelper. I changed the name from "splitted", that's why it appeared here. However, in my own PR, I've made some refactoring so that the method's gone.

Reply:
Doesn't seem so. I removed it now.